src/core/AutoRest.Core/Properties/Resources.Designer.cs

```diff
@@ -674,7 +674,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Non-HTTPS/HTTP schemes have limited support.
```

Comment:
nit: I'm assuming you have verified this is what the error message should read like